docs/reference/setup/install/docker.asciidoc

```diff
@@ -55,75 +52,25 @@ endif::[]
 
 ifeval::["{release-state}"!="unreleased"]
 
-{es} can be quickly started for development or testing use with the following command:
+To start a single-node cluster {es} for development or testing, specify 
+<<single-node-discovery,single-node discovery>> to bypass the <<bootstrap-checks,bootstrap checks>>:
 
 ["source","sh",subs="attributes"]
 --------------------------------------------
 docker run -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" {docker-image}
 --------------------------------------------
 
-Note the use of <<single-node-discovery,single-node discovery>> that allows bypassing
-the <<bootstrap-checks,bootstrap checks>> in a single-node development cluster.
-
 endif::[]
 
-[[docker-cli-run-prod-mode]]
-===== Production mode
-
-[[docker-prod-prerequisites]]
-[IMPORTANT]
-=========================
-
-The `vm.max_map_count` kernel setting needs to be set to at least `262144` for
-production use. Depending on your platform:
-
-* Linux
-+
---
-The `vm.max_map_count` setting should be set permanently in `/etc/sysctl.conf`:
-[source,sh]
---------------------------------------------
-$ grep vm.max_map_count /etc/sysctl.conf
-vm.max_map_count=262144
---------------------------------------------
-
-To apply the setting on a live system type: `sysctl -w vm.max_map_count=262144`
---
-
-* macOS with https://docs.docker.com/engine/installation/mac/#/docker-for-mac[Docker for Mac]
-+
---
-The `vm.max_map_count` setting must be set within the xhyve virtual machine:
+[[docker-compose-file]]
+==== Starting a multi-node cluster with Docker compose
 
-["source","sh"]
---------------------------------------------
-$ screen ~/Library/Containers/com.docker.docker/Data/vms/0/tty
---------------------------------------------
+The following Docker compose file brings up a three-node {es} cluster. 
+Node `es01` listens on `localhost:9200` and `es02` and `es03` talk to `es01` over a Docker network.
 
-Just press enter and configure the `sysctl` setting as you would for Linux:
-
-["source","sh"]
---------------------------------------------
-sysctl -w vm.max_map_count=262144
---------------------------------------------
---
-
-* Windows and macOS with https://www.docker.com/products/docker-toolbox[Docker Toolbox]
-+
---
-The `vm.max_map_count` setting must be set via docker-machine:
-
-["source","txt"]
---------------------------------------------
-docker-machine ssh
-sudo sysctl -w vm.max_map_count=262144
---------------------------------------------
---
-=========================
-
-The following example brings up a cluster comprising two {es} nodes.
-To bring up the cluster, use the
-<<docker-prod-cluster-composefile,`docker-compose.yml`>> and just type:
+The https://docs.docker.com/engine/tutorials/dockervolumes[Docker named volumes] 
```

Comment:
This URL now redirects to https://docs.docker.com/storage/volumes/ in case we want to use the up-to-date link.